mappings/net/minecraft/entity/ai/brain/task/LookTargetUtil.mapping

```diff
@@ -64,6 +64,10 @@ CLASS net/minecraft/unmapped/C_glrklsny net/minecraft/entity/ai/brain/task/LookT
 		ARG 1 target
 		ARG 2 speed
 		ARG 3 completionRange
+	METHOD m_twcvmjrq throwStack (Lnet/minecraft/unmapped/C_usxaxydn;Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_vgpupfxx;F)V
+		ARG 0 thrower
+		ARG 2 tossVec
```

Comment:
Why use both toss and throw?

Why not `throwVec` and `throwStrength`?

Reply:
I would say throwDirection

Reply:
good question, `throw...` would be better